### ISSUE TITLE
Python 2.4 - exceptions.AttributeError: 'module' object has no attribute 'kwdict'

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10772]
|<img alt="cphsu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| cphsu reported|
|-|-|
|Trac ID|trac#1001|
|Type|defect|
|Created|2005-04-24 22:05:29Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1001 1001
type__defect defect
reporter__cphsu cphsu
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1114380329000000 1114380329000000
changetime__1114384130000000 1114384130000000
version__ 
owner__ 
cc__exarkun cc__cphsu
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="cphsu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cphsu commented|
|-|-|

```
#!html
<pre>
At twisted\python\htmlizer.py, line 37 in printtoken

"if keyword.kwdict.has_key(token):" should be changed to "if
keyword.iskeyword(token):", since Python 2.4 has removed "kwdict"

35            self.parameters = 1
36        elif type == tokenize.NAME:
37             if keyword.kwdict.has_key(token):
38                 type = 'keyword'

ps. I found this bug while trying Nevow-0.4.1 examples with Python 2.4

Best Regards,
Kent Hsu
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
Fixed in 2.0
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
